packages/types/src/interfaces/types.ts

```diff
@@ -1,35 +1,9 @@
 // Auto-generated via `yarn polkadot-types-from-defs`, do not edit
 /* eslint-disable */
-import { AccountId, Address } from '@polkadot/types/interfaces';
-import { AnyNumber } from '@polkadot/types/types';
-import BN from 'bn.js';
-
-export {
-  Hash,
-  Block,
-  AccountId,
-  EventRecord,
-  AssetId,
-  Keys,
-  Balance,
-  Nominations,
-  Permill,
-  RuntimeDispatchInfo,
-  EraIndex,
-  Exposure,
-  StakingLedger,
-  ValidatorPrefs,
-  Forcing,
-} from '@polkadot/types/interfaces';
-export * from '@polkadot/types/types';
-
 export * from './attestation/types';
```

Comment:
Moved the additional types to the @cennznet/types..
Now we just need to copy from 
cp -rf ../api-types/src/interfaces/ packages/types/src/interfaces/
and need not worry about adding new types..they will still be available

Reply:
can you explain abit more, should this be part of the build script?

Reply:
this was concern raised by Alex.. when I copied from api-types to API project.. we got some build errors, cause auto generation, removed the additional types I added to the file.. so Alex suggested if we could me those additional types somewhere else so that when we copy, we don't need to bring that again..

Reply:
As karishma saying, the aim should be at keeping the autogenerated files intact, so we can easily regenerate and copy them across. The removed lines had previously added here manually

Reply:
Gotcha, in the long run we could generate them here in this project only, or include api-types as a proper dependency rather than copy-paste